edgedb-tokio/src/tutorial.md

```diff
@@ -0,0 +1,725 @@
+# EdgeDB Rust client tutorial
+
+# Getting started (project template repo)
```

Comment:
I'll bring in these suggestions to switch the wording though I think I'll end up turning this part into just a quick link to the repo instead of a separate section before the main "Getting started" part.